application/src/main/resources/ecc.yml

```diff
@@ -79,6 +79,12 @@ repair:
   ## The default FileBasedRepairConfiguration uses a schedule.yml file to define per-table configurations.
   ##
   provider: com.ericsson.bss.cassandra.ecchronos.application.FileBasedRepairConfiguration
+
+  ##
+  ## Specifies if tables with TWCS (TimeWindowCompactionStrategy) should be ignored for repair
+  ##
+  ignore_twcs_tables: false
```

Comment:
Nit: Have this setting after lock_type to keep the most integral settings towards the top